src/utils/entity-utils.ts

```diff
@@ -1,20 +1,23 @@
 import { Request } from 'express';
 import { Optional } from '../typing-utils/typings';
+import { IGNORE_KEY } from '../constants';
 
 const alwaysTrueFn = (item: any) => true;
 
 export const matchEntitiesByParams = (params: Record<string, string>, paramMatch: Optional<Record<string, string>>) => {
   return paramMatch ? (entity: any) => Object.entries(params).every(([param, value]) => {
     const entityKey = paramMatch[param];
-    return String(entity[entityKey]) === value;
+
+    return entityKey === IGNORE_KEY || String(entity[entityKey]) === value;
   }) : alwaysTrueFn;
 };
 
 export const matchEntitiesByBodyFilters = (body: Request['body'], filterMatch: Optional<Record<string, string>>) => {
   return filterMatch ? (entity: any) => Object.entries(body).every(([param, value]) => {
```

Comment:
for the body and for the query we might want to search over paramMatch first or the user will need to mark everything as ignored.